lib/rules/indent.js

```diff
@@ -1131,9 +1146,16 @@ module.exports = {
 
                 const tokenAfterOperator = sourceCode.getTokenAfter(operator);
 
-                offsets.ignoreToken(operator);
-                offsets.ignoreToken(tokenAfterOperator);
-                offsets.setDesiredOffset(tokenAfterOperator, operator, 0);
+
+                if (options.BinaryExpression !== BINARY_EXPRESSION_DEFAULT_INDENT) {
+                    addBlocklessNodeIndent(node);
+                    offsets.setDesiredOffset(tokenAfterOperator, operator, options.BinaryExpression);
```

Comment:
specifically, i suspect that i'm not doing the right thing here, but it's not clear to me what the right thing should be.

Reply:
I’m not familiar enough with this rule to give you much guidance off the top of my head. I’d suggest setting only:true on one case that is failing and use a debugger to step through and see what values you’re getting.